hometask4/controllers/userController.js

```diff
@@ -0,0 +1,101 @@
+import uuid from 'uuid-random';
+
+const usersMap = new Map();
+
+//read
+const _read = (req, res) => {
+	const id = req.params.id;
+	
+	const user = usersMap.get(id);
+	if (user && !user.isDeleted) {
```

Comment:
It must've been not entirely clear from my side, but the "error first" handling you fixed in the "authController.js" is also applicable for all the other controllers.
![image](https://user-images.githubusercontent.com/25229823/77852666-8bcc6100-71e8-11ea-8288-cf904e679089.png)